tests/test_course.yml

```diff
@@ -6,7 +6,7 @@ repositories:
     url: https://kubernetes-charts-incubator.storage.googleapis.com
 minimum_versions: #set minimum version requirements here
   helm: 2.8.2
-  reckoner: 0.5.0
+  reckoner: 100.1.0
```

Comment:
I am adjusting this because this test course should be deployable - it's confusing to have a "passing" test consume this file and work with this set strangely.